src/main/java/org/broadinstitute/hellbender/tools/examples/metrics/multi/ExampleMultiMetricsCollector.java

```diff
@@ -0,0 +1,144 @@
+    }
+
+    /**
+     * Process the record and create a collector args object to hold the results of any
+     * calculation for re-use across multiple levels.
+     */
+    @Override
+    protected PerUnitExampleMultiMetricsCollectorArgs makeArg(
+            final SAMRecord samRecord,
+            final ReferenceSequence refSeq) {
+        // Example collector doesn't use any cached values from the record so its a no-op.
+        return new PerUnitExampleMultiMetricsCollectorArgs();
+    }
+
+    /**
+     * Make an PerUnitExampleMultiMetricsCollector with the given arguments
+     */
+    @Override
+    protected PerUnitExampleMultiMetricsCollector makeChildCollector(
+            final String sample,
+            final String library,
+            final String readGroup) {
+        return new PerUnitExampleMultiMetricsCollector(
+                sample,
+                library,
+                readGroup);
+    }
+
+    /**
+     * Combine two ExampleMultiMetricsCollector objects and return a single ExampleMultiMetricsCollector
+     * object representing the combined results. NOTE: this implementation is destructive in that it
+     * merges the source into the target and returns the target as the combined object.
+     * @param target target destination of combined metrics. May not be null.
+     * @param source source of metrics to be combined into target. May not be null.
+     * @return single object representing the combined source and target objects
+     */
+    public ExampleMultiMetricsCollector combine(
+            final ExampleMultiMetricsCollector target,
+            final ExampleMultiMetricsCollector source) {
+        Utils.nonNull(target);
+        Utils.nonNull(source);
+        target.combine(source);
+        return target;
+    }
+
+    /**
+     * Combine two PerUnitExampleMultiMetricsCollector objects and return a single PerUnitExampleMultiMetricsCollector
+     * object representing the combined results.
+     * @param collector1 source PerUnitExampleMultiMetricsCollector. May not be null.
+     * @param collector2 target PerUnitExampleMultiMetricsCollector. May not be null.
+     * @return single PerUnitExampleMultiMetricsCollector object representing the combined source and target objects
+     */
+    public PerUnitExampleMultiMetricsCollector combineUnit(
+            final PerUnitExampleMultiMetricsCollector collector1,
+            final PerUnitExampleMultiMetricsCollector collector2) {
+        Utils.nonNull(collector1);
+        Utils.nonNull(collector2);
+        return collector1.combine(collector2);
+    }
+
+    /**
+     * Finish the metrics collection by saving any results to a metrics file.
+     * @param metricsFile a metricsFile where the collected metrics should be stored. May not be null.
+     * @param inputName the name of the input, for optional inclusion in the metrics file. May not be null.
+     * @param authHolder Authentication info for this context.
+     */
+    public void finish(
```

Comment:
It's a bit confusing that this is named `finish` since it's not an override of `finish()`.  I know it's a pattern used elsewhere though. 

Also, inputName isn't used anywhere.